Osiris/Hacks/Misc.cpp

```diff
@@ -317,42 +317,21 @@ void Misc::spectatorList() noexcept
 
 static void drawCrosshair(ImDrawList* drawList, const ImVec2& pos, ImU32 color) noexcept
 {
-    // dot
-    drawList->AddRectFilled(pos - ImVec2{ 1, 1 }, pos + ImVec2{ 2, 2 }, color & IM_COL32_A_MASK);
-    drawList->AddRectFilled(pos, pos + ImVec2{ 1, 1 }, color);
-
-    // left
-    drawList->AddRectFilled(ImVec2{ pos.x - 11, pos.y - 1 }, ImVec2{ pos.x - 3, pos.y + 2 }, color & IM_COL32_A_MASK);
-    drawList->AddRectFilled(ImVec2{ pos.x - 10, pos.y }, ImVec2{ pos.x - 4, pos.y + 1 }, color);
-
-    // right
-    drawList->AddRectFilled(ImVec2{ pos.x + 4, pos.y - 1 }, ImVec2{ pos.x + 12, pos.y + 2 }, color & IM_COL32_A_MASK);
-    drawList->AddRectFilled(ImVec2{ pos.x + 5, pos.y }, ImVec2{ pos.x + 11, pos.y + 1 }, color);
-
-    // top (left with swapped x/y offsets)
-    drawList->AddRectFilled(ImVec2{ pos.x - 1, pos.y - 11 }, ImVec2{ pos.x + 2, pos.y - 3 }, color & IM_COL32_A_MASK);
-    drawList->AddRectFilled(ImVec2{ pos.x, pos.y - 10 }, ImVec2{ pos.x + 1, pos.y - 4 }, color);
-
-    // bottom (right with swapped x/y offsets)
-    drawList->AddRectFilled(ImVec2{ pos.x - 1, pos.y + 4 }, ImVec2{ pos.x + 2, pos.y + 12 }, color & IM_COL32_A_MASK);
-    drawList->AddRectFilled(ImVec2{ pos.x, pos.y + 5 }, ImVec2{ pos.x + 1, pos.y + 11 }, color);
+    drawList->AddCircleFilled(pos, miscConfig.recoilCrosshair.thickness, color, 360);
+    drawList->AddCircle(pos, miscConfig.recoilCrosshair.thickness, IM_COL32_BLACK, 360, 0.5);
```

Comment:
**The default crosshair is almost invisible, add +1 tickness to it by default**
`Example:`

```cpp
drawList->AddCircleFilled(pos, miscConfig.recoilCrosshair.thickness + 1, color, 360);
drawList->AddCircle(pos, miscConfig.recoilCrosshair.thickness + 1, IM_COL32_BLACK, 360, 0.5);
```

Reply:
This is not necessary as it can be configured in the game